src/pages/Home/index.tsx

```diff
@@ -194,7 +198,7 @@ const HomePage = () => {
         <Col
           xs={{ span: 24, order: 2 }}
           md={{ span: 12, order: 2 }}
-          lg={6}
+          lg={!isTestNet ? 6 : 12}
```

Comment:
For example this if statement can be eliminated by providing a context object which will hold 6 in one case and 12 in the other.

Reply:
I will try to make these changes tomorrow :)

Reply:
I just realized that your suggestions only moves the `if` upwards. And imo it's worse because then you have random values centralized and you don't know what they're for, right?

Reply:
I am suggesting that any values that change between networks are kept inside a context object.
And accessed in a uniform across all networks. Every network should be described by a context object.

For example, live_context could be:
```
{
    "default_port" : 7075,
    "default_directory" : "~/Nano"
}

whereas for test network:
{
    "default_port" : 17075,
    "default_directory" : "~/NanoTest"
}

There should only be a one switch/select statement at the very top, to select the context to use.

Reply:
Lets me explain with a question, if one day we have 100 different networks, are you going to have if statements scattered around the code, to check if it is running in live, test, beta or whatever network?